k8s/bases/swe-d2cc/swe-d2cc-boundary-minio-automatic-purge.yaml

```diff
@@ -0,0 +1,36 @@
+kind: CronJob
```

Comment:
I would like to put all files in a subfolder like swe-d2cc/scripts-purge/ or something to keep it clean. We can discuss it